Frameworks/UIKit/UITextField.mm

```diff
@@ -93,18 +72,14 @@ @implementation UITextField {
     BOOL _adjustsFontSizeToFitWidth;
```

Comment:
are there any of these that we can track on the Xaml element vs. also caching them here?